### PR TITLE
Random QEMU mac address, and lower memory allocation

### PR DESCRIPTION
**Pull request by GregorioDiStefano**
- Random mac address, so we get a random client ID, allowing us to deploy many QEMU clients in docker

- Lower memory from 1G => 512M

### CODE REVIEW

**Comment by GregorioDiStefano** [comment]
@kacf

File: scripts/mender-qemu [hunk 1]
```diff
@@ -40,14 +40,15 @@ fi
 UBOOT_ELF=${UBOOT_ELF:-"${BUILDTMP}/deploy/images/vexpress-qemu/u-boot.elf"}
 VEXPRESS_SDIMG=${VEXPRESS_SDIMG:-"${BUILDTMP}/deploy/images/vexpress-qemu/${IMAGE_NAME}-vexpress-qemu.sdimg"}
 QEMU_SYSTEM_ARM=${QEMU_SYSTEM_ARM:-"${BUILDTMP}/sysroots/x86_64-linux/usr/bin/qemu-system-arm"}
+RANDOM_MAC="52:54:00$(od -txC -An -N3 /dev/urandom|tr \  :)"
```
Comment (kacf) [inline scripts/mender-qemu]: Clever!

**Comment by kacf** [comment]
I'm waiting to see if Jenkins will come back up today, but otherwise this looks good to me.

**Comment by kacf** [comment]
I'm trusting you! :-)

**Pull request merged**